src/screens/ProfileScreen/index.js

```diff
@@ -74,6 +79,7 @@ import ProfileHeader from './elements/ProfileHeader';
 import ProfilePicture from './elements/ProfilePicture';
 import ProfileTiktokScroll from './elements/ProfileTiktokScroll';
 import ImageCompressionUtils from '../../utils/image/compress';
+import {KarmaLock} from './elements/KarmaLock';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `KarmaLock` component is newly introduced. Make sure it's properly unit tested and its integration doesn't break the existing UI layout.
```diff
+ import {KarmaLock} from './elements/KarmaLock';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->